### PR TITLE
(CDAO-6035) Set context classloader for Workflow

### PR DESCRIPTION
**Pull request by chtyim**
- Set it when calling initialize and destroy.

### CODE REVIEW

**Comment by sagarkapare** [comment]
LGTM 👍

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java [hunk 1]
```diff
@@ -341,9 +350,8 @@ private WorkflowActionSpecification getActionSpecification(WorkflowActionNode no
         actionSpec = node.getActionSpecification();
         break;
       default:
-        LOG.error("Unknown Program Type '{}', Program '{}' in the Workflow.", actionInfo.getProgramType(),
-                  actionInfo.getProgramName());
-        throw new IllegalStateException("Workflow stopped without executing all tasks");
+        throw new IllegalStateException(String.format("Unknown Program Type '%s', Program '%s' in the Workflow",
```
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java:353]: nice, that error message was really misleading!

**Comment by anew** [comment]
LGTM

**Comment by chtyim** [comment]
Build passed: http://builds.cask.co/browse/CDAP-DUT4176-1

**Comment by chtyim** [comment]
Fixed commit message typo. Merging.

**Pull request merged**